Views/data2.js

```diff
@@ -0,0 +1,195 @@
+import * as React from 'react';
+import { Text, View, StyleSheet, Image } from 'react-native';
+
+export default function Data2() { 
```

Comment:
I do like these place older pieces of data but I wonder if we should migrate before deployment to a loading animation or skeleton screen?